libdevcore/IndentedWriter.h

```diff
@@ -61,7 +59,7 @@ class IndentedWriter
 		unsigned indentation;
 	};
 
-	std::vector<Line> m_lines;
+	std::vector<Line> m_lines {{std::string(), 0}};
```

Comment:
```suggestion
	std::vector<Line> m_lines{{std::string(), 0}};
```
(why should the spacing be different than for statements?)